src/coreclr/src/nativeaot/Directory.Build.props

```diff
@@ -98,4 +98,6 @@
     <AotCommonPath>$(MSBuildThisFileDirectory)Common\src\</AotCommonPath>
     <CompilerCommonPath>$(MSBuildThisFileDirectory)..\tools\Common\</CompilerCommonPath>
   </PropertyGroup>
+
+  <Import Condition="'$(MSBuildProjectName)' != 'System.Private.CoreLib'" Project="$(RepositoryEngineeringDir)versioning.targets" />
```

Comment:
Why do we exclude this for CoreLib?

Reply:
CoreCLR CoreLib also doesn't include this target and does these things "manually". Just wanted to stay in sync.